configstack/module_test.go

```diff
@@ -325,7 +325,7 @@ func TestGetTerragruntSourceForModuleErrors(t *testing.T) {
 }
 
 func mockOptionsWithSource(sourceUrl string) *options.TerragruntOptions {
-	opts := options.NewTerragruntOptionsForTest("mock-for-test.tfvars")
+	opts, _ := options.NewTerragruntOptionsForTest("mock-for-test.tfvars")
```

Comment:
I would not ignore this error. It'll lead to confusing test failures. Again, add a `t *testing.T` to this method and call `t.FatalF` in case of error.